config/_shared.json

```diff
@@ -139,5 +139,6 @@
 	},
 	"site_filter": [],
 	"theme": "default",
-	"site_name": "WordPress.com"
+	"site_name": "WordPress.com",
+	"speedcurve_id": "511719913"
```

Comment:
In Speecurve you can have multiple sites. This is the ID of the site `Wordpress.com`.

If we go to prod, we may want to have multiple sites (one per environment) to avoid mixing data.